src/components/Navbar.css

```
@@ -0,0 +1,63 @@
+* {
+    font-family: Poppins;
+
+}
```

This should go in the main css of the file or apply these changes to the body tag. Also always remember to add a fallback font family.

```suggestion
body {
font-family: 'Poppins', sans-serif;
}
```